platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureFragment.kt

```diff
@@ -87,17 +90,13 @@ open class FeatureFragment : Fragment() {
 
     private lateinit var feature: Feature
     private lateinit var foregroundResources: ForegroundResources
+
     // Public for test purposes
     lateinit var featureContainer: FeatureContainer
     private val args: FeatureFragmentArgs by navArgs()
 
     private val errorDialog: AlertDialog by lazy {
         AlertDialog.Builder(context)
-            .setPositiveButton(
-                context?.getString(R.string.button_feature_error_close)
-            ) { _, _ ->
-                close()
-            }
```

Comment:
refactored this as needed the text that pops to be send to the ErrorUploader. so moved it into the `handleError` below on the `setButton` attr